### PR TITLE
adds atmospheric gloves, small resprite of firefighter gear, repaths stupid glove paths

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
repaths a lot of gloves off /color because they were incredibly stupid
firefighter gear has gotten an update (it doesnt cover hands anymore though, you need something else)
firefighter helmets no longer hide your mask or glasses
image
fixed engine goggles starting with darkness vision
to the atmos lockers adds atmospheric gloves, a pair of thick (chunky fingers) gloves that are fireproof and fire protective, slightly shock resistant and let you fireman carry people faster.
atmospheric firefighter helmets now are a subtype of welding hardhats, you can enable a welding visor.
welding hardhats change mode with right click instead of altclick

## Why It's Good For The Game
im not a good spriter but i think this resprite makes them fit nicer with other engi equipment
lets me firefighter rp

## Changelog
:cl:
add: Atmospheric Gloves, thick gloves that are fully fireproof and fire protective and let you fireman carry people faster.
fix: fixes engine goggles starting with darkness vision
qol: firefighter helmets can now enable a welding visor
qol: welding hardhats change mode with right click instead of altclick
balance: firesuits no longer protect your hands
/:cl:

### CODE REVIEW

File: code/modules/clothing/gloves/insulated.dm [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/obj/item/clothing/gloves/color
+	dying_key = DYE_REGISTRY_GLOVES
+	greyscale_colors = null
+
+/obj/item/clothing/gloves/color/yellow
```
Comment (JohnFulpWillard) [inline code/modules/clothing/gloves/insulated.dm:5]: Did you intentionally leave ``color`` path here?

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -569,6 +569,7 @@
 		"pacman",
 		"plasma_tank",
 		"plasmaman_tank_belt",
+		"generic_tank",
```
Comment (JohnFulpWillard) [inline code/modules/research/techweb/all_nodes.dm:572]: wtf is this lol

**Comment by Serxule** [comment]
can we see the gloves tho

**Comment by private-tristan** [comment]
> (it doesnt cover hands anymore though, you need something else)

Does this apply to all firefighting gear? If so then wouldn't this make firefighting lockers mostly useless unless you already found black gloves?

File: code/modules/clothing/gloves/insulated.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/clothing/gloves/insulated.dm:5]: yes cause they are yellow

**Comment by Fikou** [comment]
> > (it doesnt cover hands anymore though, you need something else)
> 
> Does this apply to all firefighting gear? If so then wouldn't this make firefighting lockers mostly useless unless you already found black gloves?

it doesnt

File: code/modules/clothing/gloves/insulated.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/clothing/gloves/insulated.dm:5]: its stupid but whatever didnt want to repath everything int he game

File: code/modules/research/techweb/all_nodes.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/research/techweb/all_nodes.dm:572]: this is the generic gas tank. this should be a separate pr

**Comment by Fikou** [comment]
> can we see the gloves tho

press icon diff bot 2
you can also see them on the character in the firesuit

**Comment by Serxule** [comment]
> > can we see the gloves tho
> 
> press icon diff bot 2 you can also see them on the character in the firesuit

i have no idea what that means, also its pretty hard to see whats the firesuit and whats the gloves since its black on black

**Comment by san7890** [comment]
> > press icon diff bot 2 you can also see them on the character in the firesuit
> 
> i have no idea what that means, also its pretty hard to see whats the firesuit and whats the gloves since its black on black

https://github.com/tgstation/tgstation/pull/72736/checks?check_run_id=10662058876

**Review by Imaginos16 — APPROVED** [review]
Good sprites, nothing more to say.

**Review by tralezab — APPROVED** [review]

**Pull request merged**